Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -418,6 +418,23 @@ spec:
     }'
 ```
 
+## Holder Pod Deprecation
+
+Rook plans to remove CSI "holder" pods in Rook v1.16. Clusters with `csi-*plugin-holder-*` pods
+present in the Rook operator namespace must plan to set `CSI_DISABLE_HOLDER_PODS` to `"true"` after
+Rook v1.14 is installed and before v1.16 is installed by following the migration sections below.
+
+CephClusters that do not use `network.provider: multus` can follow the
```

Comment:
My confusion before was that all clusters without multus would need to go to that section. But the point is that only clusters with holder pods need to do it, right?
```suggestion
Clusters with holder pods enabled that do not use `network.provider: multus` can follow the
```

Reply:
It's mentioned immediately above that clusters with holder pods must migrate.

> Clusters with `csi-*plugin-holder-*` pods
present in the Rook operator namespace must plan to set `CSI_DISABLE_HOLDER_PODS` to `"true"` after
Rook v1.14 is installed and before v1.16 is installed by following the migration sections below.